blackjack/blackjack.py

```diff
@@ -0,0 +1,217 @@
+from game import *
+from deck import *
+from card import *
+from player import *
+from dealer import *
+from hand import *
+import os
+
+# Checks to see if the 2 initial cards are blackjack
+# (value of 21 points with 2 cards)
+
+
+def is_blackjack(card1, card2):
+    if card1.get_rank() == "A" and card2.get_rank() in ["10", "J", "Q", "K"]:
+        return True
+    elif card2.get_rank() == "A" and card1.get_rank() in ["10", "J", "Q", "K"]:
+        return True
+    else:
+        return False
+
+# validate card adds a card to a hand after doing all
+# the checks to handle the Aces and the special situations they have
+# When there is already an Ace with a value of 11 (Soft hand)
+# and that value of 11 needs to be changed to 1 (turning the hand into Hard)
+# it handles all special cases for the Ace in the hand
+
+
+def validate_card(player, card):
```

Comment:
When you have logic this nested, it makes sense to break up your function into multiple functions.